doc/faq.rst

```diff
@@ -58,6 +58,15 @@ fundamentally multi-dimensional. If your data is unstructured or
 one-dimensional, stick with pandas.
 
 
+Gotchas
+-------
+
+xarray tries hard to be self-consistent. Operations on xarray objects return
```

Comment:
I would note that the self-consistency is that the DataArray methods already return DataArrays (and the same for Dataset), i.e., it preserves the type. This isn't true for NumPy or pandas, which return scalars.

Reply:
I just noticed but numpy aggregation funcs return boxed objects that look like an ndarray

Reply:
Yes, NumPy and pandas return boxed scalar types (subclasses of `np.generic`).